circleci-orb/scripts/setup-bazel-remote-exec/index.ts

```diff
@@ -14,6 +14,12 @@ import path from 'path';
 import fs from 'fs';
 import os from 'os';
 
+/** The configuration block added to the bazelrc for configuration of remote execution on CI. */
+const bazelRcAddition = `
+build --config=remote
+build --repository_cache=${process.env['HOME']}/bazel_repository_cache
```

Comment:
I personally don't think this should live in the remote execution action to be honest. We could have a separate composite action that would setup RBE + other things.

Reply:
The composite action could then also wire up the cache action and other things

Reply:
PTAL, see if the refactor is what you were thinking.